### PR TITLE
Add server name env variable

### PR DESCRIPTION
**Pull request by fpagnoux**
Fixes openfisca/legislation-explorer#92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,14 @@ git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
 Each sub-directory has its own README.md file.
+
+## Services
+
+We use [systemd](https://wiki.debian.org/systemd) services to run the OpenFisca related applications (OpenFisca-France [web](https://api.openfisca.fr/) [APIs](https://api-test.openfisca.fr/), [legislation explorer](https://legislation.openfisca.fr/)).
+
+The service configuration files are located in the `/etc/systemd/system` directory.
```
Comment (michelbl) [inline README.md]: Do you mean one should manually copy these files to `/etc/systemd/system` ?

File: README.md [hunk 2]
```diff
@@ -14,3 +14,14 @@ git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
 Each sub-directory has its own README.md file.
+
+## Services
+
+We use [systemd](https://wiki.debian.org/systemd) services to run the OpenFisca related applications (OpenFisca-France [web](https://api.openfisca.fr/) [APIs](https://api-test.openfisca.fr/), [legislation explorer](https://legislation.openfisca.fr/)).
```
Comment (michelbl) [inline README.md]: Then why is `deploy.sh` needed ?

Comment (fpagnoux) [inline README.md]: `deploy.sh` usually updates all the packages, and restart the service.

File: README.md (hunk 1 above)
Comment (fpagnoux) [inline README.md]: Either this, or add a symlink from this directory to the source code directory.

File: README.md (hunk 2 above)
Comment (MattiSG) [inline README.md]: > “to run” → “to ensure the continued operation of” (or something along these lines)

File: README.md (hunk 1 above)
Comment (MattiSG) [inline README.md]: Suggest symlinking in the documentation rather than just replying in the PR comment :wink:

File: README.md [hunk 3]
```diff
@@ -14,3 +14,14 @@ git clone git@github.com:openfisca/openfisca-ops.git
...
+
+The service configuration files are located in the `/etc/systemd/system` directory.
+
+When you edit one of these files, run this command afterwards so that your changes are actually taken into account:
```
Comment (MattiSG) [inline README.md]: Remove “actually”.

File: README.md (hunk 2 above)
Comment (MattiSG) [inline README.md]: Specify with which OS this strategy is compatible.

**Pull request merged**